tests/integration/utils/dev-server.cjs

```diff
@@ -119,11 +119,12 @@ const startDevServer = async (options, expectFailure) => {
   // eslint-disable-next-line fp/no-loops
   for (let attempt = 1; attempt <= maxAttempts; attempt++) {
     try {
-      const { timeout, ...server } = await startServer({ ...options, expectFailure })
-      if (timeout) {
-        throw new Error(`Timed out starting dev server.\nServer Output:\n${server.output}`)
+      // do not use destruction, as we use getters which otherwise would be evaluated here
+      const devServer = await startServer({ ...options, expectFailure })
+      if (devServer.timeout) {
+        throw new Error(`Timed out starting dev server.\nServer Output:\n${devServer.output}`)
       }
-      return server
+      return devServer
```

Comment:
All the changes here werw done because we were restructuring the response of `startServer` which was cloning the object, which was evaluating the getters in the response instantly.
So it wasn't working as expected when calling `devServer.output` in the test, as it would never update, because it was evaluated as soon as the server was started.